src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -333,6 +333,16 @@ uint64_t MPIDI_OFI_index_allocator_alloc(void *_indexmap, MPL_memory_class class
 void MPIDI_OFI_index_allocator_free(void *_indexmap, uint64_t index);
 void MPIDI_OFI_index_allocator_destroy(void *_indexmap);
 
+static inline int MPIDI_OFI_index_is_valid(uint64_t index, unsigned index_bitsize)
```

Comment:
`MPL_STATIC_INLINE_PREFIX`

Reply:
Let's use `bool` as the return value.

Reply:
Since `index_bitsize` is always `MPIDI_Global.max_mr_key_size * 8`, how about drop this parameter from the function and refer to `MPIDI_Global.max_mr_key_size * 8` internally?

Reply:
I can change the return type to `bool`. I thought about passing the size as bytes rather than bits (the current), but I think the function is more general.